Configuration/TCA/Overrides/tt_content.php

```diff
@@ -4,26 +4,31 @@
     die('Access denied.');
 }
 
-\TYPO3\CMS\Extbase\Utility\ExtensionUtility::registerPlugin(
-    'Weather2',
-    'Currentweather',
-    'LLL:EXT:weather2/Resources/Private/Language/locallang_db.xlf:plugin.currentweather.title',
-);
+use TYPO3\CMS\Core\Utility\ExtensionManagementUtility;
+use TYPO3\CMS\Extbase\Utility\ExtensionUtility;
 
-\TYPO3\CMS\Extbase\Utility\ExtensionUtility::registerPlugin(
-    'Weather2',
-    'Weatheralert',
-    'LLL:EXT:weather2/Resources/Private/Language/locallang_db.xlf:plugin.weatheralert.title',
-);
+(static function (): void {
```

Comment:
Why you decided for that way? Sure, it works, but IMO we haven't any extension using that way.